### PR TITLE
Feature/mm

### PR DESCRIPTION
**Pull request by mikevanhemert**

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+image:
+  repository: 765814079306.dkr.ecr.us-east-1.amazonaws.com/mixmode
+  pullPolicy: Always
+  tag: "feature_mm"
```
Comment (bburky) [inline chart/values.yaml:4]: Do we want to use `latest` or a specific version?

**Review by bburky — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# MixMode Server 8.11
+
+The MixMode server or 'brain'.
+
+## VM OCI Image Creation
+
+The GitLab pipeline uses `virt-v2v` and `virt-customize` to convert the MixMode VM then builds an OCI container. The converted VM is saved on S3 at `naps-dev-artifact/naps-dev/mixmode/[branch or tag]` ([ex](https://s3.console.aws.amazon.com/s3/object/naps-dev-artifacts?region=us-east-1&prefix=naps-dev/mixmode/feature/mm/mixmode-8.11-sda)) and the image uploaded to ECR at `ECR \ Repositories \ mixmode \ [branch or tag]` ([ex](https://us-east-1.console.aws.amazon.com/ecr/repositories/private/765814079306/mixmode/_/image/sha256:18435cb63d84c86034e31a4c7355efda1651ebe042d507720ef92eba5da15572/details?region=us-east-1)).
```
[the remark this answers is not in the document]
Comment (bburky) [reply README.md:7]: Oh, I realized this is actually still linking to a `naps-dev/mixmode/feature/mm` URL, which is specific to this feature branch

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  --delete '/etc/cloud/cloud.cfg.d/curtin-preserve-sources.cfg' \
+  --delete '/etc/cloud/cloud.cfg.d/subiquity-disable-cloudinit-networking.cfg' \
+  --delete '/etc/netplan/*.yaml' \
+  --delete '/etc/cloud/ds-identify.cfg'  \
+  --run-command 'cloud-init clean --logs --seed'  \
+  --delete '/var/lib/cloud/'  \
+  --run-command 'sed -i '"'"'s/GRUB_CMDLINE_LINUX="\([^"]*\)"/GRUB_CMDLINE_LINUX="\1 console=tty0 console=ttyS0,115200n8"/'"'"' /etc/default/grub'  \
+  --append-line '/etc/default/grub:GRUB_TERMINAL=serial' \
+  --append-line '/etc/default/grub:GRUB_SERIAL_COMMAND="serial --speed=115200 --unit=0 --word=8 --parity=no --stop=1"' \
+  --run-command 'grub2-mkconfig -o /boot/grub2/grub.cfg'
```
[the remark this answers is not in the document]
Comment (mikevanhemert) [reply build.sh:24]: The bash script was converting, customizing, building the image, and pushing the image and running it from the pipeline seemed a bit big / obscured why or where a failure happens lest a developer try to walk through the script's output. How do you suggest integrating the pipeline with the script more cleanly?

File: chart/values.yaml (hunk 1 above)
Comment (mikevanhemert) [reply chart/values.yaml:4]: Oh shoot! This should be using a Zarf var to set :frowning: I need to clean it up. A specific version - that matches the branch - was my intent.

**Review by bburky — APPROVED** [review]

**Pull request merged**